lib/OpenQA/WebAPI/Controller/Test.pm

```diff
@@ -247,7 +247,13 @@ sub list_running_ajax {
                 progress => $job->progress_info,
             });
     }
-    $self->render(json => {data => \@running});
+    my %response = (data => \@running);
+    my $max_running = OpenQA::App->singleton->config->{scheduler}->{max_running_jobs} // -1;
```

Comment:
Why is `// -1` needed here?

Reply:
because its default if isnt set in config ?

Reply:
I'm actually not sure what happens if someone sets it to the empty string, then we would get a warning.
But I assume if it is not set at all, it would get our default from Setup.m, so I guess this is not strictly needed...

Reply:
I removed it

Reply:
Hm, some tests are now failing: https://app.circleci.com/pipelines/github/os-autoinst/openQA/12040/workflows/04e65a20-0920-4b4d-ba1b-0d7b7f4e2e3d/jobs/112482
```
Use of uninitialized value $limit in numeric ge (>=) at /home/squamata/project/lib/OpenQA/Scheduler/Model/Jobs.pm line 47.
Use of uninitialized value $limit in concatenation (.) or string at /home/squamata/project/lib/OpenQA/Scheduler/Model/Jobs.pm line 48.
Use of uninitialized value $limit in numeric ge (>=) at /home/squamata/project/lib/OpenQA/Scheduler/Model/Jobs.pm line 47.
Use of uninitialized value $limit in numeric ge (>=) at /home/squamata/project/lib/OpenQA/Scheduler/Model/Jobs.pm line 47.
Use of uninitialized value $limit in concatenation (.) or string at /home/squamata/project/lib/OpenQA/Scheduler/Model/Jobs.pm line 48.
```
I thought OpenQA::Setup would ensure the default value of -1?

Reply:
So, I added the `//=1` again.